### ISSUE TITLE
Nuclio Build Error When Trying to Setup a Custom Yolov5 Annotator

### ISSUE THREAD

**Issue by justin-larking-pk** [openvinotoolkit/cvat#4567]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
That the nuclio function would build and a custom-yolov5 function would be usable for auto-annotation

### Current Behaviour
Fails to build the function:

Error - Invalid Spec.Build.Image passed
    ...//nuclio/pkg/platform/abstract/platform.go:337

Call stack:
Docker image fields validation failed
    ...//nuclio/pkg/platform/abstract/platform.go:337
Failed to validate a function configuration
    /nuclio/pkg/platform/local/platform.go:1088
Failed to enrich and validate the updated function configuration
    /nuclio/pkg/platform/local/platform.go:204
OnAfterConfigUpdate returned error
    /nuclio/pkg/processor/build/builder.go:238
Failed to deploy function
    ...//nuclio/pkg/platform/abstract/platform.go:182

Copied the layout and function.yaml and main.py from the ultralytics/nuclio directory. Changed the labels to be that of my model, and a few other params such as name and python version. Changed the torchhub.load() to use my custom weights.

Then ran:
nuctl deploy --project-name cvat   --path cvat/serverless/pytorch/Peek/yolov5/nuclio   --volume `pwd`/serverless/common:/opt/nuclio/common

This errors out, however the yolov5 example builds just fine.

### Context
Trying to setup autolabelling for a custom dataset using a custom detector trained on some labels of the dataset.

### Your Environment
OS: Ubuntu Focal, 20.04
Docker: 20.10.14

**Comment by nmanovic** [comment]
@justin-larking-pk , did you resolve the issue?

**Comment by lqsweet** [comment]
> @justin-larking-pk , did you resolve the issue?

![2022-04-14 160514](https://user-images.githubusercontent.com/38726025/163341879-a940f524-767f-4364-ac5a-fd33d47f73e9.png)
![2022-04-14 160545](https://user-images.githubusercontent.com/38726025/163341888-2da2280b-7edf-48db-84e7-859c3b9cc2ad.png)

**Comment by songtf525** [comment]
@justin-larking-pk , did you resolve the issue?

**Comment by lqsweet** [comment]
> @justin-larking-pk , did you resolve the issue?

Many thanks, but I already solved the problem , that is because function.yaml contains invalid comma
